plugins/banners/get.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const getSchema = schema.models.banner.get;
+const idSchema = joi.reach(schema.models.banner.base, 'id');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/banners/{id}',
```

Comment:
why is this route required ?

Reply:
This is not currently required, but may be useful later on